docs/plugins/hooks.md

```diff
@@ -0,0 +1,98 @@
+# Hooks
+
+Below are details about the plugin hooks in Amphora. Plugin hooks are fired on each plugin supplied to Amphora at instantiation time. A plugin should expose a property whose name corresponds to one of the hooks below and is a function that expects the arguments detailed below.
+
+---
+
+### Routes (`routes`)
+
+A hook that gets called with the argument of the [Express Router](https://expressjs.com/en/4x/api.html#express.router) for each site in a Clay instance. This hooks allows a plugin to attach a route to a site that can then be called from a client.
+
+**Arguments:**
+  - `router`. An instance of the [Express Router](https://expressjs.com/en/4x/api.html#express.router) for a site
+
+---
+
+### Save (`save`)
+
+A hook that is called with an Array of DB operations that were just processed from Clay data structure(s) being saved.
+
+**Arguments:**
+  - `ops`: an array of DB operations
```

Comment:
Can we give an example somewhere of what one of these operations look like?

Reply:
That should be handled in a different doc about how to hook up whichever key/val store you want.